### PR TITLE
signal fragments and the background fragments/cards for upgrade study

### PR DESCRIPTION
**Pull request by bobaekim**
Excited Lepton Search for a HL-LHC Yellow Paper
The excited lepton signal is split into the e* and mu* channels, so it is 10k events for each e* mass point and 10k events for each mu* mass point.
In case of background sample, the card generation scripts are adjusted copies of https://github.com/jfernan2/genproductions/tree/c0898316e58cf83a96f21679bd77a110ade8797b/bin/MadGraph5_aMCatNLO/cards/production/14TeV/ZATo2LA01j_5f_NLO_FXFX and https://github.com/cms-sw/genproductions/tree/mg26x/bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZATo2LA01j_5f_pta130_NLO_FXFX
And Zgamma with photon pt > 130 GeV, 500 GeV are uploaded.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/14TeV/ZATo2LA01j_5f_pta130_NLO_FXFX/Hadronizer_ZATo2LA01j_pta130_14TeV_madgraph_cff.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import FWCore.ParameterSet.Config as cms
+
+externalLHEProducer = cms.EDProducer("ExternalLHEProducer",
+    args = cms.vstring('/afs/cern.ch/user/b/bokim/work/public/background/ZATo2LA01j_5f_pta130_NLO_FXFX_slc6_amd64_gcc481_CMSSW_7_1_30_tarball.tar.xz'),
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/14TeV/ZATo2LA01j_5f_pta130_NLO_FXFX/Hadronizer_ZATo2LA01j_pta130_14TeV_madgraph_cff.py]: why is the file pointing to an afs location? only cvmfs should be used

File: bin/MadGraph5_aMCatNLO/cards/production/14TeV/Estar_EG_L10000_m1000_to_m6500_pythia8/Estar_EG_L10000_m1000_14TeV_pythia8.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import FWCore.ParameterSet.Config as cms
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/14TeV/Estar_EG_L10000_m1000_to_m6500_pythia8/Estar_EG_L10000_m1000_14TeV_pythia8.py]: why are these cfg put under this folder?
they should go under
https://github.com/cms-sw/genproductions/tree/master/python/ThirteenTeV/Estar

Comment (bobaekim) [reply bin/MadGraph5_aMCatNLO/cards/production/14TeV/Estar_EG_L10000_m1000_to_m6500_pythia8/Estar_EG_L10000_m1000_14TeV_pythia8.py]: Dear perrozzi,

I have 2 questions. 
My cfg files are for 14 TeV. Can I put files under Thirteen TeV not FourteenTeV?
And about moving gridpack, can I make directory as below?
/eos/cms/store/group/phys_generator/gridpacks/slc6_amd64_gcc481/14TeV/madgraph/V5_2.4.2/
or do I follow as instruction like /store/group/phys_generator/cvmfs/gridpacks/<your scram architecture>/13TeV/<your generator>/<your version>/ ?
Thank you in advance.

Best regards,
Bobae

**Comment by qliphy** [comment]
The python fragment is not supposed to uploaded here. Please remove them.

The relevant area is:
https://github.com/cms-sw/cmssw/tree/master/Configuration/Generator/python
But usually it is just pasted by MC contact in McM.

**Pull request closed without merging**